### PR TITLE
WebUSB build fix

### PR DESCRIPTION
**Pull request by pelikhan**
Fixing "storage" library.

### CODE REVIEW

File: libs/storage/pxt.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "storage",
-    "description": "USB Pen-drive support and flash storage - beta",
+    "description": "USB Pen-drive support and flash storage",
     "files": [
```
Comment (mmoskal) [inline libs/storage/pxt.json]: Doesn't this need DEVICE USB as well?

**Comment by pelikhan** [comment]
DEVICE_USB is set in the codal target. I guess it does not hurt to add it (BLE nano does not have that define).

**Comment by mmoskal** [comment]
Ble nano doesn't have usb. I guess you're right storage could work without usb

**Comment by mmoskal** [comment]
Usb is currently only supported on samd21. STM 32 was close iirc and d51 should be easy

**Pull request merged**